README.md

```diff
@@ -6,76 +6,70 @@
-```
-@import url("https://fonts.googleapis.com/css?family=Lato:300,400,700");
-```
-
-### Standalone style usage
-If your project doesn't use yarn or npm for dependencies and you're only looking for masterclass css / style files, they can be imported manually by downloading the repository on github.  This will give you access to styles for buttons, the grid, typography, and more.  Save the contents of the `/dist/styles/scss/` folder to your own project, then import!
-
-**Note**: If your project doesn't have sass processing, there is a plain css file you can import here:
-
-```
-/dist/styles/css/mc-components.css
+```bash
+yarn add mc-components
 ```
 
-### Using a component
+### Components
 ```javascript
 import { Button } from 'mc-components'
 
 const Container = () =>
   <Button primary>Hello</Button>
 ```
 
-# Contributing
-1. Clone the repository
-2. Install dependencies with `yarn install`
-3. Create a branch off `develop`
-4. Start storybook: `yarn start`
+### Styles
 
-All changes are hot-reloaded and you'll be able to see components being modified live as you work.
+The style library must also be imported manually into your root `SCSS` file.
 
-If you want to develop in `mc-components` and see your work compiled live in your own project (like the masterclass repo), you can run the following commands:
+```scss
+@import 'mc-components/dist/styles/scss/index'
+```
 
-In `mc-components`: (this creates a symlink to use in your project's repo, and also will let you hot-reload changes to styles and components in mc-components)
+We use Lato as our primary font for headings and body text.  You can import it in your CSS below, or import it into your asset pipeline.
 
+```scss
+@import url("https://fonts.googleapis.com/css?family=Lato:300,400,700")
 ```
+
+# Development
+
+`mc-components` utilizes [Storybook](https://storybook.js.org/), which provides a living version of the documentation for all components.
+
+```bash
 yarn install
+yarn start
+```
+
+If you want to develop in `mc-components` and see changes compiled into your own project you can utilize `yarn link` and the `dev` script:
+
+```bash
+# mc-components
 yarn link
 yarn dev
+
+# other project
+yarn link mc-components
 ```
 
-In your own project (eg MasterClass) directory:
+To get back to an officially released version, run this in your project:
 
+```bash
+yarn unlink mc-components
+yarn install
 ```
-yarn link mc-components
-yarn webpack-dev
-```
 
-(Run `yarn unlink "mc-components"` in your project's repo when you are done developing and want to unlink, and then run `yarn install` to reinstall the version of mc-components you were using previously)
+To update the [live documentation](https://yankaindustries.github.io/mc-components), run the following command from the root directory:
+
+```bash
+yarn deploy
+```
 
-## Submit your code
+## PRs Welcome
   - Create a PR with your changes
```

Comment:
What if they don't know how to create a PR

Reply:
they can upload with FTP or email them to me